### DISCUSSION TITLE
What is the supported frequency range in Sionna Ray Tracing?

### DISCUSSION THREAD

**Discussion by mustafafu** [NVlabs/sionna#322] [Q&A] [answered]
Hi all,
Is there a limit to the supported frequency range in Sionna Ray tracing?
More specifically, I am interested in support in upper 6 GHz band. Is this band supported in ray tracing?
I understand some ITU materials are only defined up to 10 GHz, therefore for those materials I assume the max frequency is 10 GHz. So U6G band should be supported.  Is this assumption valid?

For other related questions, I see people asked about mmWave frequencies. If mmWave freqs. are interest to someone else, please see links below.
https://github.com/NVlabs/sionna/discussions/210
https://github.com/NVlabs/sionna/discussions/238

Thanks a lot.

**Comment by jhoydis** [comment, marked as answer]
Hi,

No there is no upper limit to the frequency and, yes, the upper 6 GHz is supported.
The only issue that you might encounter is that different ITU materials support different frequency ranges. However, you can simply assign different material properties from the literature for this case or assume that the models are valid outside this range.
